src/coreclr/src/gc/unix/gcenv.unix.cpp

```diff
@@ -28,6 +28,12 @@
 #undef min
 #undef max
 
+#if __has_cpp_attribute(fallthrough)
+#define FALLTHROUGH [[fallthrough]];
```

Comment:
Can this be `#define __fallthrough [[fallthrough]]` ?

Reply:
Symbols starting with underscore are reserved for the compiler and the compiler warns. We have the warning disabled for coreclr, but not for PAL. Disabling it for coreclr would be complicated due to all of the SAL stuff. But maybe we should move coreclr to use FALLTHROUGH too so that it is uniform.

Reply:
Actually - a small correction - I didn't hit the warning in PAL, but in libraries minipal. I had __fallthrough in PAL before and then I've unified PAL and libraries minipal on the uppercase version.

Reply:
I also tried to use the attribute directly without the macro, but that broke GCC build as it was not supported until GCC 7

Reply:
Yes, it would be nice to use same macro everywhere. I see number of other projects using `#define FALLTHROUGH` : https://github.com/search?p=2&q=%22%23define+fallthrough%22&type=Code.